src/team_lib/mhba/xccl_mhba_lib.h

```diff
@@ -73,6 +73,9 @@ extern xccl_team_lib_mhba_t xccl_team_lib_mhba;
 #define MAX_MSG_SIZE 128 // HW transpose unit is limited to element size
 #define MAX_STRIDED_ENTRIES 55 // from limit of NIC memory - Sergey Gorenko's email
 #define MAX_BLOCK_SIZE 64 // from limit of Transpose unit capabilities
+#define NUM_DCI_QPS 16
+#define RC_DC_LIMIT 128
```

Comment:
why not a parameter?

Reply:
that will also be a fixed number we will find out in real-time measurements in the future, in UCX it's 200 for example.

Reply:
do you mean adding that as a run parameter, like Transpose?

Reply:
ill add as param